### PR TITLE
chore: update readme and files structure

### PR DESCRIPTION
**Pull request by AragonCodes**
## Description
Following our agreements from the "fe-unified-pipele" slack channel, we will use this repo to test Jr devs.

Please review: 
1. README
2. `src`
3. `src/tests`

## TODOs
- [x] define `contact` email (maybe frontend-hiring@adjust.com?)
- [x] define `frontend-hiring-github-user` GitHub account to evaluate candidates submissions
- [x] Add shared account credentials to 1Password

### CODE REVIEW

**Review by Sc4ramouche — APPROVED** [review]
Looks good to me, thank you! 👍 
We can also specify "Add shared account credentials to 1Password" or something like this as a "TODO" 🙂

**Review by BrunoAzzi — APPROVED** [review]

File: src/tests/solution.test.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { renderData, prepareData } from "../src/solution";
+import { renderData, prepareData } from "../solution";
```
Comment (BrunoAzzi) [inline src/tests/solution.test.js:1]: We need to move or delete the original file.

**Review by paul-pro — APPROVED** [review]
We should also ask someone from the team to follow both instructions and solve this challenge

File: src/tests/solution.test.js (hunk 1 above)
Comment (AragonCodes) [reply src/tests/solution.test.js:1]: What to do you mean to erase the original file? 🤔

Comment (BrunoAzzi) [reply src/tests/solution.test.js:1]: Oh! nvmd! Just noticed we changed `solution.test.js` location. Its ok 👍🏽

**Comment by AragonCodes** [comment]
Please take a look 🙏🏻  @moktin @plachintas @lemanchester @PetrusStarken

**Review by plachintas — APPROVED** [review]

**Pull request merged**